foosball/scoreboard/main.fbp

```diff
@@ -32,6 +32,9 @@ Score(ScoreLed)
 ScoreSwitcher(switcher/int:keep_state=true)
 RedWin(RedWinLed)
 YellowWin(YellowWinLed)
+P1Sound(piezo-speaker:tune="aabbccdd")
+P2Sound(piezo-speaker:tune="aabbccdd")
+
```

Comment:
Actually, I would like to use different sounds, but I didn't test this sample with the piezo-speaker, so I don't know what sounds better.